views/js/qtiCreator/renderers/choices/InlineChoice.js

```diff
@@ -1 +1,37 @@
-//not used
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2014 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
```

Comment:
> 1. Have issue with addChoice button

Is it FE issue?
> 2. Math in inline choice is not rendered properly in item preview. Issue with item runner?Should be a) fixed in scope of this ticket or b) should be created separate ticket and this PR should be merged in working branch not develop.

We have a separate ticket for TR, so I changed the destination branch so this will not be merged to develop.
> 3. Can't preview test
> 4. Can't publish test

To fix the rendering we need to add changes in QTI-SDK cause now it supports only text variable for inline choice options https://github.com/oat-sa/qti-sdk/blob/legacy/qtism/data/storage/xml/marshalling/InlineChoiceMarshaller.php#L57. I fixed this on my local environment by replacing `TextOrVariableCollection` with `FlowStaticCollection`  for `inlineChoice` content type. But we need to ask @julien-sebire is it possible to add such changes in QTI-SDK and how we may release such breaking change.

Reply:
> Is it FE issue?

Yes, probably, I'll take a look

Reply:
Now should be fixed